### PR TITLE
Added Fake script to build release version

### PR DESCRIPTION
**Pull request by Slesa**

### CODE REVIEW

**Comment by MrCube42** [comment]
Works! 👍

File: build.cmd [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+SET FAKE_PATH=packages\build\FAKE\tools\Fake.exe
+SET Platform=
+
+IF [%1]==[] (
+    "%FAKE_PATH%" "build.fsx" "Default" 
+) ELSE (
+    "%FAKE_PATH%" "build.fsx" %* 
+) 
+
+if defined TEAMCITY_PROJECT_NAME goto Quit
```
Comment (MrCube42) [inline build.cmd:27]: Do we need this?

File: build.cmd [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+rem Loop the build script.
```
Comment (MrCube42) [inline build.cmd:29]: Why should one loop the build script? It is meant to be run alongside developing?

File: build.fsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+open Fake
+open System.IO
+
+
+let binDir = "bin"
+let buildDir = binDir </> "build"
+
+let cwd = System.IO.Directory.GetCurrentDirectory()
+Log "Wotrking in ..." [ cwd ]
```
Comment (MrCube42) [inline build.fsx]: typo

File: paket.lock [hunk 4]
```diff
@@ -0,0 +1,108 @@
+CONTENT: NONE
```
Comment (MrCube42) [inline paket.lock:1]: What exactly is this file good for? Does it tell the exact version ranges needed? What is the difference to packet.dependencies?

**Review by MrCube42 — CHANGES_REQUESTED** [review]
Could you please comment on (or fix) my findings

File: build.cmd (hunk 1 above)
Comment (Slesa) [reply build.cmd:27]: Depends on the build server this file will build on. Or if we keep the loop at all.

File: build.cmd (hunk 2 above)
Comment (Slesa) [reply build.cmd:29]: It is very helpful when you change the build.fsx. You simply hit enter. Additional, the build script will not try to resolve paket.exe again and again.

File: paket.lock (hunk 4 above)
Comment (Slesa) [reply paket.lock:1]: It is the corresponding file for the paket.dependency. "paket install" will create it, but the build script assumes this already has been done.

File: build.fsx (hunk 3 above)
Comment (Slesa) [reply build.fsx]: Both lines have to be removed. Was debugging.

**Review by MrCube42 — APPROVED** [review]

**Pull request merged**